crates/nu-command/src/system/run_external.rs

```diff
@@ -548,7 +548,7 @@ impl ExternalCommand {
         }
     }
 
-    fn create_process(
+    pub(crate) fn create_process(
```

Comment:
that might be a very dumb `rust` question, but i do not see why you need to make that function public :thinking: 

it's not used anywhere in your changes, right? :open_mouth:

Reply:
it still looks public, you changed it from `pub(crate)` to `pub`, right?